test/presubmit-tests-sdk.sh

```diff
@@ -22,11 +22,9 @@ python3 -m pip install -r "$source_root/sdk/python/requirements.txt"
 python3 -m pip install --upgrade protobuf
 
 # Testing the component authoring library
-component_lib=/tmp/component_lib
```

Comment:
It looks like you've effectively removed this test suite.
What is the main reason for removal of this test?
The tests here were verifying the architectural layering and module isolation.
As per the development guidelines, we avoid circular references.

These are permitted dependencies:

`kfp.dsl` can depend on `kfp.components`
`kfp.compiler` can depend on `kfp.dsl` and `kfp.components`

Could you please add TODO item to remove circular references and restore the module decoupling?

Reply:
We plan to move away from relative imports. This is working as intended.